stdlib/public/Darwin/Foundation/String.swift

```diff
@@ -96,3 +96,23 @@ extension Substring : _ObjectiveCBridgeable {
 }
 
 extension String: CVarArg {}
+
+/*
+ This is on NSObject so that the stdlib can call it in StringBridge.swift
+ without having to synthesize a receiver (e.g. lookup a class or allocate)
+ */
+private extension NSObject {
+  @_effects(releasenone)
+  @objc(newTaggedNSStringWithASCIIBytes_:length_:)
```

Comment:
This selector has to start with "new" to get swiftc to not autorelease the return value

Reply:
Are the trailing underscores intended to convey privateness without screwing up the `new` prefix?

Reply:
yup